src/super_gradients/common/sg_loggers/base_sg_logger.py

```diff
@@ -102,27 +114,39 @@ def _make_dir(self):
 
     @multi_process_safe
     def _init_log_file(self):
-        time_string = time.strftime('%b%d_%H_%M_%S', time.localtime())
-        self.log_file_path = f'{self._local_dir}/log_{time_string}.txt'
+        time_string = time.strftime("%m_%d_%H_%M_%S", time.localtime())
+        # There are two log files, since the regular log_file_path used for `manual` logging of configs/other info
+        # While the log_full_file_path serves as a sink for all SG-related events.
+        # We can combine these two by changing _write_to_log_file implementation to call getLogger().info(...)
+        # if that's would be ok
+        # TODO: Check with Ofri
+        self.log_file_path = f"{self._local_dir}/log_{time_string}.txt"
+        self.log_full_file_path = f"{self._local_dir}/log_full_{time_string}.txt"
+        AutoLoggerConfig.log_everything_to_file(self.log_full_file_path)
+        AutoLoggerConfig.add_console_logging(logging.WARNING)
```

Comment:
There are some  INFO logs that I think are nice to see by default.
Why not set it by default to INFO, and add the possibility to set the level with an env variable, like it is done here ? : https://github.com/Deci-AI/super-gradients/blob/0fa438b73bf0a3812910245c2e1ee1aefb50391c/src/super_gradients/common/abstractions/abstract_logger.py#L9